cmd/service.go

```diff
@@ -102,7 +102,11 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			log.Println("Exiting.")
+			go func() {
+				time.Sleep(time.Millisecond * 600)
+				log.Println("press ^C again")
```

Comment:
`press ^C again`

Just saying this will make the user think "why is it asking me to press ^C again?"

You can put the message as:
`Waiting for active connections to terminate, press ^C again to stop.`